src/gui/accountsettings.cpp

```diff
@@ -58,6 +57,8 @@
 
 namespace OCC {
 
+Q_LOGGING_CATEGORY(lcAccountSettings, "gui.account.settings", QtInfoMsg)
```

Comment:
We don't have any other gui.settings yet, so not sure. I however doubt that people will tweak the rules in `gui.` except for the specific part they work on.